docker/config/crontabber.env

```diff
@@ -0,0 +1,10 @@
+# Crontabber configuration file covering behavior configuration for
+# crontabber
+
+# NOTE: This file shouldn't contain any environment-specific or secret
+# configuration.
+
+# ----------------------------------------------------------------------------------
+# from socorro/crontabber
+# ----------------------------------------------------------------------------------
+crontabber.class-BugzillaCronApp.days_into_past=0
```

Comment:
Is just just an example or does that need to be set for things to work?
`0` is the default already.